content/en/docs/reference/kubectl/quick-guide.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: kubectl Cheat Sheet
+title: kubectl Quick Guide
```

Comment:
nit: Is this supposed to read:
Quick Start or Quick Start Guide

Reply:
"Quick Start Guide" looks good. Let me update it. THanks!

Reply:
A word of caution. Readers may expect a “Quick Start Guide” to explain how to download and use the tool. That is commonly what a Quick Start Guide pages explains.

I'd actually prefer something like Quick Reference.

Reply:
I agree with @sftim, a "Quick Start" or "Quick Start Guide" would be understood on how to get a running Kubernetes cluster up and running quickly

I don't mind "kubectl Quick Reference" or how about "kubectl Common Examples" -- "Common Examples" doesn't flow as smooth as "Quick Reference" but it describes the page clearer

Reply:
The latest commit has suggestions from this conversation. Please review it and let me know if more can be done from myside. THanks for taking time everyone!